.gitignore

```diff
@@ -31,4 +31,5 @@ override.tf.json
 # */**/.terraform/
 *.tfstate.*
 .terraform/
-.vscode/
\ No newline at end of file
+.vscode/
+**/*.terraform.lock.hcl
```

Comment:
We want the lock file and it should be updated w/ `tf init -upgrade` every release. We need to verify the lock file is valid as well at least eventually.